readme.md

```diff
@@ -220,27 +223,32 @@ Then, go to [http://localhost:5000/](http://localhost:5000) and you should see a
   ```
 
 * **Your Answer:**
+* Creating a url friendly id that is short enough for a url character limit.
 
 * **Question:** What routes are currently available to you as part of this application?
 
 * **Your Answer:**
+* vegetables and vegetables/:ids
 
 * **Question:** Look for `helpers.validate` in the `app.js` file. What is this and how does it work?
 
 * **Your Answer:**
+* It appears to be a validation check for the two "required" inputs or responses of name and price.
 
 * **Question:** Try creating a new vegetable. Then, try restarting your server. What happens to the data you posted and why?
 
 * **Your Answer:**
+I was having issues with getting anything to post to state.  But considering that it was a state if you refresh the server I would assume state would be saved.  
```

Comment:
Since we never saved the data to a file or database it exists only as a javascript object which is lost when the server restarts.